qcodes/instrument_drivers/AimTTi/AimTTi_PL601P_channels.py

```diff
@@ -0,0 +1,311 @@
+        """
+        This is the private function that ensures that the output is switched
+        off before changing the current range, as pointed out by the instrument
+        manual.
+        """
+        channel_id = self.channel
+        self.output(False)
+        self.write(f'IRANGE{channel_id} {val}')
+
+    def increment_volt_by_step_size(self) -> None:
+        """
+        A bound method that increases the voltage output of the corresponding
+        channel by an amount the step size set by the user via ``volt_step_size``
+        parameter.
+        """
+        channel_id = self.channel
+        self.write(f'INCV{channel_id}')
+        # Clear the cache.
+        _ = self._get_voltage_value()
+
+    def decrement_volt_by_step_size(self) -> None:
+        """
+        A bound method that decreases the voltage output of the corresponding
+        channel by an amount the step size set by the user via ``volt_step_size``
+        parameter.
+        """
+        channel_id = self.channel
+        self.write(f'DECV{channel_id}')
+         # Clear the cache.
+        _ = self._get_voltage_value()
+
+    def increment_curr_by_step_size(self) -> None:
+        """
+        A bound method that increases the current output of the corresponding
+        channel by an amount the step size set by the user via ``curr_step_size``
+        parameter.
+        """
+        channel_id = self.channel
+        self.write(f'INCI{channel_id}')
+         # Clear the cache.
+        _ = self._get_current_value()
+
+    def decrement_curr_by_step_size(self) -> None:
+        """
+        A bound method that decreases the current output of the corresponding
+        channel by an amount the step size set by the user via ``curr_step_size``
+        parameter.
+        """
+        channel_id = self.channel
+        self.write(f'DECI{channel_id}')
+         # Clear the cache.
+        _ = self._get_current_value()
+
+    def save_setup(self, slote: int) -> None:
+        """
+        A bound function that saves the output setup to the internal
+        store specified by the numbers 0-9.
+        """
+        if not slote in self.set_up_store_slots:
+            raise RuntimeError("Slote number should be an integer between"
+                               "0 adn 9.")
+
+        channel_id = self.channel
+        self.write(f'SAV{channel_id} {slote}')
+
+    def load_setup(self, slote: int) -> None:
+        """
+        A bound function that loadss the output setup from the internal
+        store specified by the numbers 0-9.
+        """
+        if not slote in self.set_up_store_slots:
+            raise RuntimeError("Slote number should be an integer between"
+                               "0 adn 9.")
+
+        channel_id = self.channel
+        self.write(f'RCL{channel_id} {slote}')
+        # Update snapshot after load.
+        _ = self.snapshot(update=True)
+
+    def set_damping(self, val: int=0) -> None:
```

Comment:
Could `damping` not be a parameter? It looks to always have a well-defined state, and it seems relevant for the user to be able to query the state.

Okay, just read the notebook: 
> Finally, the current meter averaging can be turned on and off remotely. As there is no remote query for the status, user should observe the "Meter Average" signal light. 

That sucks. But a parameter with `get_cmd=None` would still be preferable, I think.

Reply:
Even if this should remain a function, I don't think there should be a default value for `val`. How is it obvious that `set_damping()` switches the damping off and not on? "Explicit is better than implicit" 🙂

Reply:
I think you are right that there should not be a default value, I removed it. However, it should stay as a function due to in fact that how the instrument works. For this and, for example, increment and decrement by step size properties, I have first created parameters which failed. They only work if they are methods of the class to which we pass the corresponding values as arguments.